examples/tuners/random_nas_tuner/random_nas_tuner.py

```diff
@@ -7,7 +7,9 @@ def random_archi_generator(nas_ss, random_state):
     '''
     chosen_archi = {}
     print("zql: nas search space: ", nas_ss)
-    for block_name, block in nas_ss.items():
+    for block_name, block_value in nas_ss.items():
+        assert block_value['_type'] == "mutable_layer", "Random NAS Tuner only receives NAS search space"
```

Comment:
why not point out we need "mutable_layer" key here.

Reply:
I have modify the info to explicitly pointed this out